articles/integrations/azure-api-management/configure-auth0.md

```diff
@@ -0,0 +1,73 @@
+# Configure Auth0
+
+To use Auth0 as an OAuth 2.0 authorization server, you'll need to execute the following set-up steps:
+
+1. Create an Auth0 API and Non Interactive Client.
+2. Create a Connection to store your users.
+3. Create a user so that you can test your integration when you've finished setting it up.
+
+### Step 1: Create an API and Non Interactive Client
+
+An API is an entity that represents an external resource that's capable of accepting and responding to requests made by clients. You'll need to create an [Auth0 API](/apis) using the Management Dashboard to represent the API managed by Azure's API Management Service that you want secured by Auth0.
```

Comment:
That's very nice (explaining why the user needs to create an API), maybe add also something about the non interactive client (which we don't explain what it is)?